concat-css.php

```diff
@@ -212,14 +208,3 @@ function __set( $key, $value ) {
 	}
 }
 
-function page_optimize_css_concat_init() {
-	global $wp_styles;
-
-	if ( page_optimize_should_concat_css() ) {
-		$wp_styles = new Page_Optimize_CSS_Concat( $wp_styles );
-		$wp_styles->allow_gzip_compression = ALLOW_GZIP_COMPRESSION;
-	}
-}
-
-add_action( 'init', 'page_optimize_css_concat_init' );
```

Comment:
Instead of having different initialization timing with `plugins_loaded` in the main module and `init` in the concat modules, let's simplify and have everything driven by a single place by a single action. This makes loading logic readable in one place and thus easier to understand.